src/client/components/vision/camera/view_model.ts

```diff
@@ -102,8 +103,11 @@ export class CameraViewModel {
     if (this.model.visualmesh && this.model.draw.visualmesh) {
       scene.add(this.visualmesh(this.model.visualmesh))
     }
-    this.model.draw.balls && this.model.balls.forEach(ball => scene.add(this.ball(ball)))
-    this.model.draw.goals && this.model.goals.forEach(goal => scene.add(this.goal(goal)))
+    if (this.model.greenhorizon && this.model.draw.greenhorizon) {
+      scene.add(this.greenhorizon(this.model.greenhorizon))
+    }
+    this.model.balls.forEach(ball => scene.add(this.ball(ball)))
```

Comment:
Regression here, you can't disable drawing balls

Reply:
Everyone wants to see the balls anyone! Especially the robot!